### PR TITLE
Reorganize documentation

### PR DESCRIPTION
**Pull request by ssbarnea**
- move examples into standalone file which is included in builded doc
- put link from project readme to documentation site
- include few basic badges in project readme

### CODE REVIEW

File: docs/examples.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+### Using Ansible runtime
```
Comment (webknjaz) [inline docs/examples.md:1]: Just make it a python file and literal—include it instead.

File: README.md [hunk 2]
```diff
@@ -1,46 +1,11 @@
 # ansible-compat
```
Comment (webknjaz) [inline README.md:1]: I think I'd want the readme to be partially included into the docs front page to avoid duplication as much as possible.

File: docs/index.md [hunk 3]
```diff
@@ -17,4 +22,5 @@
 * :ref:`genindex`
 * :ref:`modindex`
 * :ref:`search`
+
```
Comment (webknjaz) [inline docs/index.md:25]: Why is this necessary?

File: README.md (hunk 2 above)
Comment (ssbarnea) [reply README.md:1]: We cannot really include it because for few reasons:
* it would include phrase that does not make much sense, pointing to the location of official documentation (would be kind recursive to display it on the official docs page)
* not sure if displaying badges is desirable in published doc but if you think so, ok
* header indentation/level would mismatch and we would get an error while building

I make it easier to progress on that just push extra fixes to that branch. I will avoid forced pushes. I have no strong feeling around how we organize it.

Comment (webknjaz) [reply README.md:1]: That's why I said *partially*. There's start-at, start-after and so on. You should only reuse parts that makes sense to reuse.

Comment (ssbarnea) [reply README.md:1]: There is nothing reusable inside root `README.md` file.
